app/routes/lending/components/LendingRequest.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Card, Flex, Icon } from '@webkom/lego-bricks';
+import cx from 'classnames';
+import { useState } from 'react';
+import styles from './LendingRequest.css';
+
+export enum status {
+  PENDING,
+  APPROVED,
+  DENIED,
+}
```

Comment:
I see you removed the other enum duplicate (which included my suggestion here) and I'm pretty sure that breaks a couple of things

```suggestion
export enum Status {
  PENDING = "pending",
  APPROVED = "approved",
  DENIED = "denied",
}
```

Reply:
I dont see what that breaks, better to just have the enum one place. Also does it matter to have `PENDING = "pending"` instead of just `PENDING`?

Reply:
<img width="147" alt="image" src="https://github.com/webkom/lego-webapp/assets/69514187/2d07d646-f3c6-48ce-b0cd-ebc7f52327a9">